drivers/audio/Kconfig

```diff
@@ -185,7 +185,7 @@ config WM8776_MSG_PRIO
 
 config WM8776_WORKER_STACKSIZE
 	int "WM8776 worker thread stack size"
-	default 768
```

Comment:
This is a horrendous increase and very WRONG!
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.